crates/types/src/id_util.rs

```diff
@@ -51,7 +52,7 @@ prefixed_ids! {
 }
 
 impl IdSchemeVersion {
-    const fn as_char(&self) -> char {
+    pub fn as_char(&self) -> char {
```

Comment:
needed? I don't see why this can't remain const.

Reply:
accident